.github/workflows/integration.yml

```diff
@@ -237,37 +238,32 @@ jobs:
   test:
     name: Launch Test
     runs-on: ubuntu-20.04
-    needs: configure
+    needs: [configure, build]
     if: needs.configure.outputs.ok-to-continue == 'true' && github.event.pull_request.draft == false
     steps:
 
-    - name: Set up Go 1.16
-      uses: actions/setup-go@v2
-      with:
-        go-version: 1.16
-      id: go
-
     - name: Check out code into the Go module directory
       uses: actions/checkout@v2
       with:
         ref: "${{ needs.configure.outputs.commit_ref }}"
         repository: "${{ needs.configure.outputs.repo-name }}"
         persist-credentials: false
 
-    - name: Install Kubebuilder
-      run: |
-        version=2.3.1 # latest stable version
-        arch=amd64
-        curl -L -O "https://github.com/kubernetes-sigs/kubebuilder/releases/download/v${version}/kubebuilder_${version}_linux_${arch}.tar.gz"
-        tar -zxvf kubebuilder_${version}_linux_${arch}.tar.gz
-        mv kubebuilder_${version}_linux_${arch} kubebuilder && sudo mv kubebuilder /usr/local/
-
-    - name: Get dependencies for Test execution
-      run: |
-        go get -u github.com/ory/go-acc
+    - name: Login to DockerHub
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKER_USERNAME }}
+        password: ${{ secrets.DOCKER_PASSWORD }}
 
-    - name: Launch Test
-      run: go-acc ./... --ignore liqo/test/e2e
+    - name: Launch Test (Docker Container)
+      uses: addnab/docker-run-action@v3
+      with:
+        options: |
```

Comment:
I think we need the `--cap-add=NET_ADMIN` option to interact with the network stack.